sources/au/act/statewide.json

```diff
@@ -10,9 +10,9 @@
     "layers": {
         "addresses": [
             {
-                "name": "state",
-                "data": "http://actmapi.act.gov.au/datadownload/Shapefile/MGA94_55/ACT_data.zip",
-                "website": "http://app.actmapi.act.gov.au/download.html",
+                "name": "ACTGOV Addresses",
```

Comment:
It doesn't like the spaces you added in the names. This could likely be fixed with some correct escaping in the script or by switching the names back to what they were.

Reply:
oops I realised now the documentation says alphanumeric + underscores.